Dockerfile

```diff
@@ -68,6 +68,9 @@ RUN \
         unzip \
         vim-common \
         wget \
+    && echo 'Installing CMake' >&2 && \
+    apt-get -y install \
+        cmake \
```

Comment:
I would prefer to only add this one to the list as was done in #55 I do not see a reason to do an extra `apt-get install`